locksmith/package.json

```diff
@@ -9,7 +9,7 @@
     "predev": "NODE_ENV=development npm run db:migrate",
     "dev": "tsc-watch --onSuccess 'node build/server.js' --onFailure 'echo Beep! Compilation Failed'",
     "pretest": "NODE_ENV=test sequelize db:migrate",
-    "test": "jest --forceExit",
+    "test": "jest  --forceExit --runInBand",
```

Comment:
Do you need this in this PR?

Reply:
It would be good to get it out with this change, this should remove issues of the test db running into locks while these tests in particular clean up prior to running